components/script/dom/webidls/HTMLElement.webidl

```diff
@@ -17,10 +17,16 @@ interface HTMLElement : Element {
   readonly attribute DOMStringMap dataset;
 
   // microdata
-  //         attribute boolean itemScope;
+  //           attribute boolean itemScope;
+  //           attribute DOMString itemProp;
```

Comment:
Why are these commented out?

Reply:
I think these lines were already there. We don't need these comments, I can delete them